code/modules/paperwork/paper_biscuit.dm

```diff
@@ -0,0 +1,125 @@
+/obj/item/folder/biscuit
+	name = "\proper biscuit card"
+	desc = "An biscuit card. To reach contents you need to crack it open. Has label which says <b>DO NOT DIGEST</b>."
+	icon_state = "paperbiscuit"
+	bg_color = "#ffffff"
+	w_class = WEIGHT_CLASS_TINY
+	max_integrity = 130
+	drop_sound = 'sound/items/handling/disk_drop.ogg'
+	pickup_sound = 'sound/items/handling/disk_pickup.ogg'
+	/// Is biscuit cracked open or not?
+	var/cracked = FALSE
+
+/obj/item/folder/biscuit/suicide_act(mob/living/user)
+	user.visible_message(span_suicide("[user] tries to eat the 'biscuit'! [user.p_theyre()] trying to commit suicide!"))
+	return TOXLOSS
+
+/obj/item/folder/biscuit/update_overlays()
+	. = ..()
+	if(contents.len) //This is to prevent the not-sealed biscuit to have the folder_paper overlay when it gets sealed
+		. -= "folder_paper"
+		if(cracked) //Shows overlay only when it has content and is cracked open
+			. += "paperbiscuit_paper"
+
+///Checks if the biscuit has been already cracked. If its not then it dipsplays "unopened!" ballon alert. If it is cracked then it lets the code continue.
+/obj/item/folder/biscuit/proc/crack_check(mob/user)
+	if (cracked)
+		return TRUE
+	balloon_alert(user, "unopened!")
+	return FALSE
+
+//All next is done so you can't reach contents, or put any new contents when its not cracked open
+/obj/item/folder/biscuit/remove_item(obj/item/item, mob/user)
+	if (!crack_check(user))
+		return
+
+	return ..()
+
+/obj/item/folder/biscuit/attack_hand(mob/user, list/modifiers)
+	if (LAZYACCESS(modifiers, RIGHT_CLICK) && !crack_check(user))
+		return
+
+	return ..()
+
+/obj/item/folder/biscuit/attackby(obj/item/weapon, mob/user, params)
+	if (is_type_in_typecache(weapon, folder_insertables) && !crack_check(user))
+		return
+
+	return ..()
+
+/obj/item/folder/biscuit/attack_self(mob/user)
+	add_fingerprint(user)
+	if (!cracked)
+		if (tgui_alert(user, "Do you want to crack it open? You cannot close it back.", "Biscuit card", list("Yes", "No")) == "Yes")
```

Comment:
```suggestion
		if (tgui_alert(user, "Do you want to crack it open? You cannot uncrack it.", "Biscuit card", list("Yes", "No")) == "Yes")
```

verb consistency. you could also turn it into an early return like so:

```suggestion
		if (tgui_alert(user, "Do you want to crack it open? You cannot uncrack it.", "Biscuit card", list("Yes", "No")) != "Yes")
			return
```

so we can de-indent the rest of the code and make it look a bit neater, especially since we don't need the `else` case anymore (this just replaces the if/else with something equivalent)